elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/scaladsl/ElasticsearchFlow.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.elasticsearch.scaladsl
+
+import akka.NotUsed
+import akka.stream.alpakka.elasticsearch.{ElasticsearchFlowStage, ElasticsearchSinkSettings, IncomingMessage}
+import akka.stream.scaladsl.Flow
+import org.elasticsearch.client.{Response, RestClient}
+import spray.json.{DefaultJsonProtocol, JsObject, JsonWriter}
+
+object ElasticsearchFlow {
+
+  /**
```

Comment:
It's unclear what the exact semantics of the flow are. How many responses do you get, one per source element or one per batch?